sklearn/metrics/_pairwise_distances_reduction/_dispatcher.py

```diff
@@ -509,13 +509,13 @@ def compute(
             The weights applied over the `labels` of `Y` when computing the
             weighted mode of the labels.
 
-        class_membership : ndarray
+        y_labels : ndarray
             An array containing the index of the class membership of the
             associated samples in `Y`. This is used in labeling `X`.
 
-        unique_classes : ndarray
+        labels : ndarray
```

Comment:
I know I gave this suggestion, but it is still kind of confusing. What about:

- `y_labels` -> `Y_labels`: Since there are the labels for upper case `Y`.
- `labels` -> `unique_Y_labels`: Directly connect it to the `Y_labels` parameter.

Reply:
I do prefer the inclusion of `unique` in the latter. For the prior, I have no strong preference. I'll add those on 👍